.ci/install_runtime.sh

```diff
@@ -77,8 +79,9 @@ if [ "$USE_VSOCK" == "yes" ]; then
 	fi
 fi
 
-echo "Add runtime as a new/default Docker runtime. Docker version \"$(docker --version)\" could change according to updates."
-docker_options="-D --add-runtime kata-runtime=/usr/local/bin/kata-runtime"
-
-echo "Add kata-runtime as a new/default Docker runtime."
-"${cidir}/../cmd/container-manager/manage_ctr_mgr.sh" docker configure -r kata-runtime -f
+if [ "$FIRECRACKER" != true ]; then
+	echo "Add runtime as a new/default Docker runtime. Docker version \"$(docker --version)\" could change according to updates."
+	docker_options="-D --add-runtime kata-runtime=/usr/local/bin/kata-runtime"
+	echo "Add kata-runtime as a new/default Docker runtime."
+	"${cidir}/../cmd/container-manager/manage_ctr_mgr.sh" docker configure -r kata-runtime -f
+fi
```

Comment:
Forgot to say, how about displaying a message for the `else` case explaining that the runtime will not be setup?